flake.nix

```diff
@@ -1,7 +1,7 @@
 {
   description = "NixOps: a tool for deploying to [NixOS](https://nixos.org) machines in a network or the cloud";
 
-  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs-channels/nixos-unstable";
```

Comment:
Why `nixpkgs-channels`? I thought this was deprecated and added to `nixpkgs` proper long ago.

Reply:
Force of habit.